cloudflare-integration.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: 'Connect your Cloudflare Worker with Xata in one click'
+description: 'Connecting a Cloudflare Worker with Xata is now possible from the Cloudflare dashboard.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/xata-workers.png
+  alt: Xata + Cloudflare
+author: Alexis Rico
+date: 12-12-2023
+tags: ['cloudflare', 'serverless', 'database', 'integration']
+published: true
+slug: cloudflare-integration
+---
+
+Cloudflare Workers is a serverless platform that empowers developers to run code at the edge of the network. This means running code closer to users, resulting in faster response times and lower latency. Cloudflare Workers also offers a secure environment, ensuring code isolation from other users on the same server.
+
+We are thrilled to announce a collaboration with Cloudflare, enabling a one-click integration directly from the Cloudflare dashboard. This user-friendly solution eliminates the need for developers to create an API key and manually configure their database connection.
+
+<ArticleImage
+  src="https://raw.githubusercontent.com/xataio/mdx-blog/cloudflare/images/cloudflare-integration-dashboard.png"
+  width="874"
+  height="509"
+  alt="Cloudflare Integration Image"
+  caption="Cloudflare dashboard with the Xata integration"
+  position="center"
+/>
+
+Simply visit the Cloudflare dashboard, navigate to the "Integrations" tab, and find the Xata integration. Click on the "Add integration" button, and a wizard will guide you through the process of connecting your Cloudflare Worker with Xata.
```

Comment:
This is a small nit, but I believe the integrations tab needs to enabled. I'm not sure if it's there by default.

https://developers.cloudflare.com/workers/databases/native-integrations/